### ISSUE TITLE
Command lines for training Protonet model

### ISSUE THREAD

**Issue by gabrielhuang** [Sha-Lab/FEAT#27]
Hi, thanks for sharing your code.

Could you provide command lines for training the Protonet models ? Or are the hyperparameters exacty the same as for FEAT ?

Thanks !

**Comment by Han-Jia** [comment]
Hi,

For ProtoNet w/ ResNet could be learned using the learning rate 0.0001, the other parameters are the same as FEAT.

**Comment by gabrielhuang** [comment]
Thank you, I will try that !
